identity/src/main/java/com/android/identity/Util.java

```diff
@@ -139,14 +139,14 @@ private Util() {
     static @NonNull
     byte[] fromHex(@NonNull String stringWithHex) {
         int stringLength = stringWithHex.length();
-        if ((stringLength & 1) != 0) {
-            throw new IllegalArgumentException("Invalid length of hex string");
+        if ((stringLength % 2) != 0) {
```

Comment:
Why is this any better? Aren't they the same thing?

Reply:
Yeah they do exactly the same thing (and may even be compiled to the same code), I just felt that % 2 is the more idiomatic / canonical way to test for oddness. I'm happy to change it back if you prefer?

Reply:
This just feels highly subjective. I read them exactly the same, but maybe my brain's been warped by firmware? @davidz25 should tie break us :)

Reply:
The non-subjective perspective is that & 1 requires thinking about how numbers are represented in twos complement, whereas % 2 only requires thinking about them as numbers.

I'm happy to change it back (sorry, it was just a drive-by change to begin with), it's just that if I change it in this commit then github will lose the comment history, or if I do it in a follow-on commit then that will be a tiny commit and will show up in git history. What's your preference?